src/index.html

```diff
@@ -17,16 +17,28 @@
     />
   </head>
   <body>
-    <input
-      type="text"
-      data-qa="keypress"
-      placeholder="Try “Los Angeles“"
-    />
+    <form
+      class="search-bar primary-search-bar"
+      data-qa="big"
+    >
+      <input
+        class="search-bar-input search-bar-input--primary"
+        type="text"
+        data-qa="keypress"
+        placeholder="Try “Los Angeles“"
+      />
+    </form>
 
-    <input
-      type="text"
-      data-qa="hover"
-      placeholder="Try “Los Angeles“"
-    />
+    <form
+      class="search-bar secondary-search-bar"
+      data-qa="small"
```

Comment:
Same comment as above for `secondary-search-bar`. A more descriptive name like `additional-search-bar` would be better.